.github/workflows/ubuntu-master.yml

```diff
@@ -30,4 +31,4 @@ jobs:
         env:
           TARANTOOL_SERVER_USER: root
           TARANTOOL_SERVER_GROUP: root
-        run: mvn -B verify --file pom.xml
+        run: ./mvnw -B verify
```

Comment:
Please separate the changes in CI regarding usage of the wrapper into a new PR. When the team agrees on it, we'll merge both of the PRs here and in the driver with the wrapper in CI.